packages/world/src/Constants.sol

```diff
@@ -23,26 +23,27 @@ uint256 constant RESPAWN_ORE_BLOCK_RANGE = 10;
 // ------------------------------------------------------------
 // Values To Tune
 // ------------------------------------------------------------
-uint128 constant MAX_PLAYER_ENERGY = 1_000_000;
-uint128 constant PLAYER_ENERGY_DRAIN_RATE = 10;
+uint128 constant SCALE_FACTOR = 10 ** 14;
+uint128 constant MAX_PLAYER_ENERGY = 8174 * SCALE_FACTOR;
+uint128 constant PLAYER_ENERGY_DRAIN_RATE = 0.01351521164 * 10 ** 14;
 
-uint128 constant MACHINE_ENERGY_DRAIN_RATE = 100;
+uint128 constant MACHINE_ENERGY_DRAIN_RATE = 0.00007966149167 * 10 ** 14;
 
-uint128 constant SMART_CHEST_ENERGY_COST = 100;
+uint128 constant SMART_CHEST_ENERGY_COST = 100 * SCALE_FACTOR;
 
-uint128 constant BUILD_ENERGY_COST = 50;
-uint128 constant MINE_ENERGY_COST = 50;
-uint128 constant HIT_ENERGY_COST = 50;
-uint128 constant TILL_ENERGY_COST = 50;
-uint128 constant CRAFT_ENERGY_COST = 50;
-uint128 constant MOVE_ENERGY_COST = 5;
+uint128 constant BUILD_ENERGY_COST = 81 * SCALE_FACTOR;
+uint128 constant MINE_ENERGY_COST = 81 * SCALE_FACTOR;
+uint128 constant HIT_ENERGY_COST = 81 * SCALE_FACTOR;
+uint128 constant TILL_ENERGY_COST = 81 * SCALE_FACTOR;
+uint128 constant CRAFT_ENERGY_COST = 81 * SCALE_FACTOR;
+uint128 constant MOVE_ENERGY_COST = 0.2554375 * 10 ** 14;
```

Comment:
do we want to just hardcode the values so it is consistent? it is weird that some use scale factor and some don't